### ISSUE TITLE
Sovol sv07 profiles for SuperSlicer

### ISSUE THREAD

**Issue by giovannirco** [TomW1605/SV07_Klipper_config#1]
Hey man, thank you very much for you configs! May I ask, do you have profiles for the slicers such as SuperSlicer or PrusaSlicer?

**Comment by TomW1605** [comment]
yeah i have SuperSlicer profiles, they are based on the SV06 profiles added to Pruser slicer (the PR you commented on as well) with some very small changes. they are not fast as at the moment as for me quality is better than speed so i havnt bothered to tune them for speed. ill upload them here when i get home from work. i have a few layer heights but i have only tested the 0.2mm one since thats all i need at the moment

**Comment by giovannirco** [comment]
thank you very much! Yeah I am aiming on print quality as well, my printer didnt arrive yet and I couldnt find much info on the internet about it. I wanted to have everything ready to play around with it when it arrives.

**Comment by TomW1605** [comment]
yeah thats why i posted this, i spent 2 days trying to find the config and couldent so wanted to make it easy for others in the same situation

**Comment by TomW1605** [comment]
the config i ended up getting whas the whole controller image and included a link to this github repo, might also be of some help (it also includes things like the klipper screen config+code changes and other custom stuff they did)

https://github.com/MS1987/SOVOL_MKDEB

**Comment by giovannirco** [comment]
hey mate, dont you want to push your .cfg to [klipper ](https://github.com/Klipper3d/klipper)repo?

My printer will arrive next week and if you open a PR I can comment/"approve" the cfg, plus it gives the chance for more people to see it.

you can refer to this doc https://github.com/Klipper3d/klipper/blob/master/docs/Example_Configs.md

And here is a [similar PR ](https://github.com/Klipper3d/klipper/pull/6056)for sv06

**Comment by TomW1605** [comment]
thats a good idea, i have some more recent changes i want to add to the github then i will look into it.

also sorry for the delay on adding the super slicer config, i forgot to do it yesterday but i have uploaded them now
